tests/integration/suite/test_multi_cluster_app.py

```diff
@@ -614,6 +615,48 @@ def test_mcapp_rollback_validation(admin_mc, admin_pc, custom_catalog,
     assert e.value.error.message == 'rancher max version exceeded'
 
 
+def test_perform_mca_action_read_only(admin_mc, admin_pc, remove_resource,
+                                      user_mc):
+    client = admin_mc.client
+    project = admin_pc.project
+    user = user_mc
+    remove_resource(user)
+
+    ns = admin_pc.cluster.client.create_namespace(
+        name=random_str(),
+        projectId=project.id)
+    remove_resource(ns)
+
+    prtb = client.create_project_role_template_binding(
+        name="prtb-" + random_str(),
+        userId=user.user.id,
+        projectId=project.id,
+        roleTemplateId="read-only")
+    remove_resource(prtb)
+
+    wait_until_available(user.client, project)
+
+    mcapp_name = random_str()
+    mcapp_user_read_only = "local://" + user.user.id,
+    mcapp = client.create_multi_cluster_app(
+        name=mcapp_name,
+        templateVersionId="cattle-global-data:library-wordpress-1.0.5",
```

Comment:
Wordpress ?

Reply:
This was the templateVersionId used from the other tests. @dramich suggested using ```docker-registry``` (less bloat), but I kept receiving an 500 HTTP error code. Need to double check this.

Reply:
Fixed.